### PR TITLE
Update run-testnet-SP-node.md

### PR DESCRIPTION
**Pull request by maxnodereal**
Update how to submit proposals

### CODE REVIEW

**Review by j75689 — APPROVED** [review]

File: docs/guide/storage-provider/run-book/run-testnet-SP-node.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ You can execute this command to query on-chain parameters. `./gnfd q sp params -
 The SP needs to initiate an on-chain proposal that specifies the Msg information to be automatically executed after the vote is approved. In this case, the Msg is `MsgCreateStorageProvider`. It's worth noting that the deposit tokens needs to be greater than the minimum deposit tokens specified on the chain.
 
 ```shell
-./build/bin/gnfd tx gov submit-proposal path/to/create_sp.json --from operator --keyring-backend os --node https://gnfd-testnet-fullnode-tendermint-us.bnbchain.org:443
+./build/bin/gnfd tx gov submit-proposal path/to/create_sp.json --from {funding_address} --keyring-backend os --node https://gnfd-testnet-fullnode-tendermint-us.bnbchain.org:443
```
Comment (sysvm) [inline docs/guide/storage-provider/run-book/run-testnet-SP-node.md:221]: It's like that I can use operator_address to submit proposal successfully, so both them can be used?

File: docs/guide/storage-provider/run-book/run-testnet-SP-node.md [hunk 2]
```diff
@@ -261,7 +261,7 @@ $ cat ./create_sp.json
 Each proposal needs to have enough tokens deposited to enter the voting stage.
 
 ```shell
-./build/bin/gnfd tx gov deposit {proposal_id} 1BNB --from {operator_address} --keyring-backend os --node https://gnfd-testnet-fullnode-tendermint-us.bnbchain.org:443
+./build/bin/gnfd tx gov deposit {proposal_id} 1BNB --from {funding_address} --keyring-backend os --node https://gnfd-testnet-fullnode-tendermint-us.bnbchain.org:443
```
Comment (sysvm) [inline docs/guide/storage-provider/run-book/run-testnet-SP-node.md:264]: ditto

File: docs/guide/storage-provider/run-book/run-testnet-SP-node.md (hunk 1 above)
Comment (KeefeL) [reply docs/guide/storage-provider/run-book/run-testnet-SP-node.md:221]: Yes, either operator address or funding address can submit this proposal, however the funding address should do the grant, we will also add a one step command for these two steps, so use funding address here is fine.

**Review by KeefeL — APPROVED** [review]

File: docs/guide/storage-provider/run-book/run-testnet-SP-node.md (hunk 1 above)
Comment (sysvm) [reply docs/guide/storage-provider/run-book/run-testnet-SP-node.md:221]: ok

**Pull request merged**